### PR TITLE
Add estimated finish Time next to "charge time left"

### PR DESCRIPTION
**Pull request by Pazekal90**
Solution for issue: https://github.com/EVNotify/EVNotifyPWA/issues/13

Bildschirmfoto 2019-05-29 um 17 25 31

I copied the existing function chargingTimeLeft(). Maybe we can globalize some constants to deduplicate some code there.

### CODE REVIEW

File: src/components/Dashboard.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@
                     <v-icon color="teal">schedule</v-icon>
                   </v-list-tile-action>
                   <v-list-tile-content>
-                    <v-list-tile-title>{{ chargingTimeLeft}} h</v-list-tile-title>
+                    <v-list-tile-title>{{ chargingTimeLeft }} h / {{ finishTime }}</v-list-tile-title>
```
Comment (GPlay97) [inline src/components/Dashboard.vue]: Would be better, if time is in brackets

00:37 h (14:38) for example

Comment (GPlay97) [inline src/components/Dashboard.vue]: Would be better, if time is in brackets

00:37 h (14:38) for example

File: src/components/Dashboard.vue [hunk 2]
```diff
@@ -186,6 +186,19 @@
         const duration = this.$root.MomentJS.duration(parseFloat(decimalTime)).asMilliseconds();
         return this.$root.MomentJS().startOf('day').add(duration, 'minutes').format('m:ss');
       },
+      finishTime() {
```
Comment (GPlay97) [inline src/components/Dashboard.vue]: As mentioned, it should be a general function (put them in `utils` folder).

Should get the car, socDisplay, socBMS, batteryPower (maybe instead just all the syncData).

The other usage should then also be fetched from the utils.

File: src/components/Dashboard.vue (hunk 1 above)
Comment (Pazekal90) [reply src/components/Dashboard.vue]: done this

File: src/components/Dashboard.vue (hunk 2 above)
Comment (Pazekal90) [reply src/components/Dashboard.vue]: Ok, done. Please have a look on my commit 8c96624. Added the file and new general function. Changed the functions calls in dashboard.vue.

Comment (GPlay97) [reply src/components/Dashboard.vue]: Wow, thank you :+1: :tada: 

Will test it later and look at it :)

**Comment by Pazekal90** [comment]
@GPlay97 Have you already reviewed some of my changes? Do want me to open a new Pull Request to get rid of conflicts?

**Comment by GPlay97** [comment]
> @GPlay97 Have you already reviewed some of my changes? Do want me to open a new Pull Request to get rid of conflicts?

Hey.

I have fixed the conflict for you :)

Will review it now and merge it afterwards (will deploy it later then).

Thank you!

**Pull request merged**